components/content-blocks/TableGroup/index.js

```diff
@@ -0,0 +1,32 @@
+import Container from "@/components/layout/Container";
+import { Tab } from "@headlessui/react";
```

Comment:
Time for me to take a closer look at this library! 😁

Reply:
Oh man, putting together the tabs was so simple with this library. ✨